zkstack_cli/crates/zkstack/src/commands/chain/gateway/migrate_to_gateway_calldata.rs

```diff
@@ -338,15 +338,26 @@ pub async fn run(shell: &Shell, params: MigrateToGatewayCalldataArgs) -> anyhow:
                 );
                 // It is the expected case, it will be handled later in the file
             }
-            GatewayMigrationProgressState::PendingManualFinalization => {
+            GatewayMigrationProgressState::PendingManualFinalization
+            | GatewayMigrationProgressState::AwaitingFinalization => {
                 unreachable!("`GatewayMigrationProgressState::PendingManualFinalization` should not be returned for migration to Gateway")
             }
-            _ => {
-                let msg = message_for_gateway_migration_progress_state(
+            GatewayMigrationProgressState::NotStarted
+            | GatewayMigrationProgressState::NotificationSent
+            | GatewayMigrationProgressState::NotificationReceived(_) => {
+                anyhow::bail!(
```

Comment:
in other places we dont use bail when notifying about another state, please either use `logger::info` or modify other places too